input/fsh/who-co-Observations.fsh

```diff
@@ -0,0 +1,196 @@
+Title: "WHO CO Outcome Patcourse ICU"
+Description: "WHO CO Outcome Patcourse ICU Observation"
+* code = $LNC#95420-6
+* value[x] only CodeableConcept
+* valueCodeableConcept 1..1
+* valueCodeableConcept from WhoCrValueSetYesNoUnk
+
+Profile: WhoCoObservationOutcomePatcourseVent
+Parent: Observation
+Id: who-co-observation-outcome-patcourse-vent
+Title: "WHO CO Outcome Patcourse Vent"
+Description: "WHO CO Outcome Patcourse Vent Observation"
+* code = $LNC#96539-2
+* value[x] only CodeableConcept
+* valueCodeableConcept 1..1
+* valueCodeableConcept from WhoCrValueSetYesNoUnk
+
+Profile: WhoCoObservationOutcomePatcourseEcmo
+Parent: Observation
+Id: who-co-observation-outcome-patcourse-ecmo
+Title: "WHO CO Outcome Patcourse Ecmo"
+Description: "WHO CO Outcome Patcourse Ecmo Observation"
+* code = $LNC#96540-0
+* value[x] only CodeableConcept
+* valueCodeableConcept 1..1
+* valueCodeableConcept from WhoCrValueSetYesNoUnk
+
+Profile: WhoCoObservationOutcomePatcourseStatus
+Parent: Observation
+Id: who-co-observation-outcome-patcourse-status
+Title: "WHO CO Outcome Patcourse Status"
+Description: "WHO CO Outcome Patcourse Status Observation"
+* code = $LNC#91541-3
+* value[x] only CodeableConcept
+* valueCodeableConcept 1..1
+* valueCodeableConcept from WhoCoValueSetHealthOutcome
+
+Profile: WhoCoObservationOutcomePatcourseStatusOther
+Parent: Observation
+Id: who-co-observation-outcome-patcourse-status-other
+Title: "WHO CO Outcome Patcourse Status Other"
+Description: "WHO CO Outcome Patcourse Status Other Observation"
+* value[x] only string
+* valueString 0..1
+
+Profile: WhoCoObservationOutcomeLabResult
+Parent: Observation
+Id: who-co-observation-outcome-lab-result
+Title: "WHO CO Outcome Lab Result"
+Description: "WHO CO Outcome Lab Result Observation"
+* code = $LNC#96552-5
+* value[x] only CodeableConcept
+* valueCodeableConcept 0..1
+* valueCodeableConcept from WhoCoValueSetPositiveNegativeUnknown
+* effective[x] only dateTime
+* effectiveDateTime 1..1
+
+Profile: WhoCoObservationOutcomeContactsFollowed
+Parent: Observation
+Id: who-co-observation-outcome-contacts-followed
+Title: "WHO CO Outcome Contacts Followed"
+Description: "WHO CO Outcome Contacts Followed Observation"
+* code = $LNC#96551-7
+* value[x] only integer
+* valueInteger 1..1
+* note 0..*
+* note.text 1..1
+
+/*
+ * Examples 
+ */
+
+Instance: WhoCoObservationOutcomeAsympExample
+InstanceOf: WhoCoObservationOutcomeAsymp
+Usage: #example
+Title: "WHO CO Outcome Asymp Example"
+Description: "Developed symptoms after time of specimen collection"
+* status = #final
+* valueCodeableConcept = $cs-v2-0136#Y
+* effectiveDateTime = "2021-05-13"
+
+Instance: WhoCoObservationOutcomeAsympNoEffectiveDateExample
+InstanceOf: WhoCoObservationOutcomeAsymp
+Usage: #example
+Title: "WHO CO Outcome Asymp Example"
+Description: "Developed symptoms after time of specimen collection example without effective date"
+* status = #final
+* valueCodeableConcept = $cs-v2-0136#Y
```

Comment:
In a more realistic case, I think the asymp date is only left out when the patient says "no".
But this is pedantic

```suggestion
Instance: WhoCoObservationOutcomeAsympNoEffectiveDateExample
InstanceOf: WhoCoObservationOutcomeAsymp
Usage: #example
Title: "WHO CO Outcome Asymp Example"
Description: "Did not develop symptoms after time of specimen collection example (no effective date)"
* status = #final
* valueCodeableConcept = $cs-v2-0136#N
```